tests/cli/cli_test.go

```diff
@@ -89,7 +92,7 @@ func run(t testing.TB, commands []Command) {
 
 		b, err := cli.Command(args...).CombinedOutput()
 		t.Log(fmt.Sprintf("\n$ %s\n%s", cmd.Command, string(b)))
-		if err != nil {
+		if !shouldError && err != nil {
```

Comment:
I like this way better, I think the CLI command is exiting with "exit status 1" error vs that error message.

```
--- FAIL: TestDeploy_CommitMessageRequired (0.11s)
	cli_test.go:93:
		$ deploy remind101/acme-inc
		error: A message is required for this action, please run again with '-m'.

	cli_test.go:96: Expected exit status 1, got error: A message is required for this action, please run again with '-m'.
```

It worked before because I was just comparing the output